ddd-koans/Value Object & Entity/Draft/src/Bar.kt

```diff
@@ -0,0 +1 @@
+class Bar(private val ordinal: Int, private val timeSignature: TimeSignature, private val notes: Notes)
```

Comment:
bars rather have `number` than `ordinal`, worth checking that it's a positive integer

Reply:
I did not intend to create full class implementation, just the type and properties.

Reply:
As it comes to number and ordinal, I don't like `number` as it does not imply any kind of order and Bars are identify by order, if I got this one right. Maybe index then?

Reply:
maybe `index`, it's hard to say as in musicians language there is no clear name for it, rather sth like `let's play from 44th bar`, maybe it's and order in the song...hard to say, let's keep it as it is